From 326909baf81a638d51fa8be1d8227518784f5cc4 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 14 Sep 2021 12:18:25 +0200
Subject: [PATCH] Configure: do not tweak mips cflags

This conflicts with mips machine definitons from yocto,
e.g.
| Error: -mips3 conflicts with the other architecture options, which imply -mips64r2

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 Configure | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Configure b/Configure
index 821e680..0387a74 100755
--- a/Configure
+++ b/Configure
@@ -1422,16 +1422,6 @@ if ($target =~ /^mingw/ && `$config{CC} --target-help 2>&1` =~ m/-mno-cygwin/m)
         push @{$config{shared_ldflag}}, "-mno-cygwin";
         }
 
-if ($target =~ /linux.*-mips/ && !$disabled{asm}
-        && !grep { $_ !~ /-m(ips|arch=)/ } (@{$config{CFLAGS}})) {
-        # minimally required architecture flags for assembly modules
-        my $value;
-        $value = '-mips2' if ($target =~ /mips32/);
-        $value = '-mips3' if ($target =~ /mips64/);
-        unshift @{$config{cflags}}, $value;
-        unshift @{$config{cxxflags}}, $value if $config{CXX};
-}
-
 # If threads aren't disabled, check how possible they are
 unless ($disabled{threads}) {
     if ($auto_threads) {
